actionpack/test/dispatch/routing_test.rb

```diff
@@ -2668,7 +2668,7 @@ def test_only_should_be_read_from_scope
         namespace :only do
           resources :clubs do
             resources :players
-            resource  :chairman
+            resource  :chairman, only: [:show]
```

Comment:
Again, this test is not testing what it should be testing. Maybe changing `:index` to `:edit`?